src/vite/plugin.tsx

```diff
@@ -21,18 +22,21 @@ type RemixViteConfig = {
   client?: Partial<RdtClientConfig>;
   server?: DevToolsServerConfig,
   pluginDir?: string;
-  includeInProd?: boolean
+  includeInProd?: boolean;
+  appDirectory?: string;
 };
  
 export const defineRdtConfig = (config: RemixViteConfig) =>  config
  
 export const remixDevTools: (args?:RemixViteConfig) => Plugin[] = (args) => {
   const serverConfig = args?.server || {};
   const clientConfig = args?.client || {};
+  const appDirectory = args?.appDirectory ?? 'app';
 //  const pluginDir = args?.pluginDir || undefined;
   const include = args?.includeInProd??false;
 //  const plugins = pluginDir ? processPlugins(pluginDir) : [];
  // const pluginNames = plugins.map((p) => p.name);
+  let rootDir = process.cwd();
```

Comment:
also I'm not sure if this is required or not, this would probably break the deployments as well as not all env have process defined.